spark/spark-1.6/pom.xml

```diff
@@ -9,13 +9,12 @@
 		<relativePath>../../pom.xml</relativePath>
 	</parent>
 
-	<artifactId>hivemall-spark</artifactId>
-	<name>Hivemall on Spark</name>
+	<artifactId>hivemall-spark_2.11</artifactId>
```

Comment:
why `2.11?` hivemall-spark-v1 and hivemall-spark-v2 might be good for artifact id.